lib/taskana-core/src/test/java/acceptance/workbasket/CreateWorkbasketAccTest.java

```diff
@@ -47,6 +47,7 @@ public void testCreateWorkbasket()
         workbasket.setName("Megabasket");
         workbasket.setType(WorkbasketType.GROUP);
         workbasket.setOrgLevel1("company");
+        workbasket.setMarkedForDeletion(false);
```

Comment:
this is the default already, isn't it?

Reply:
I remove all "workbasket.setMarkedForDeletion(false);"